mkdocs.yml

```diff
@@ -10,6 +10,7 @@ nav:
     - Deploying your first model: 'getting-started.md'
```

Comment:
Possibly need to update the getting started page section:
```
Training and deploying
Now that you've reached this point, you'll have provisioned the following infrastructure into Azure:

The MLFlow experiment tracker and model registry.
Seldon for model deployment and serving.
A ZenML server. This example uses ZenML for defining and orchestrating the training and deployment pipelines.
Kubernetes. This has two roles: firstly, it's where the training workload actually runs, and secondly it's the deployment environment for all of the above components.
```
with DVC?

Reply:
I think a mention on this page is worthwhile, but I'm not sure it's worth the bloat DVC documentation would bring here if someone just wants to get up and running. What do you think?

Reply:
Possibly link to the larger section on DVC?

Reply:
Yeah that's what I went for in the end.